keyboards/geminate60/readme.md

```diff
@@ -0,0 +1,3 @@
+# 60% Keyboards
```

Comment:
Needs to follow template <https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md>